plugins/usd/module/vtkF3DUSDImporter.cxx

```diff
@@ -539,8 +540,22 @@ class vtkF3DUSDImporter::vtkInternals
     pxr::UsdSkelBakeSkinning(this->Stage->Traverse());
     this->Stage->Save();
 
-    vtkNew<vtkMatrix4x4> identity;
-    this->ImportNode(renderer, this->Stage->GetPseudoRoot(), pxr::SdfPath("/"), identity);
+    vtkNew<vtkMatrix4x4> rootTransform;
+
+    pxr::TfToken up = pxr::UsdGeomGetStageUpAxis(this->Stage);
+
+    if (up == pxr::UsdGeomTokens->z)
+    {
+      vtkWarningWithObjectMacro(
+        renderer, << "This USD file up axis is +Z, internally rotate it to align with +Y axis");
```

Comment:
use F3DLog

Reply:
Can we use F3DLog from VTK classes?

Reply:
Hmmm it seems so... Should we?

Reply:
F3DLog has been created for logging from VTKExtensions class in F3D.

vtk macro should not be used as they were not supposed to be shown to the user. (they are currently shown in debug mode because it sadly appears to be needed).

Reply:
We need the VTK extension for that, see #1115